src/test/scala/firrtlTests/AnnotationTests.scala

```diff
@@ -445,6 +445,15 @@ class AnnotationTests extends AnnotationSpec with Matchers {
       anno("foo", mod = "DeadExt"), anno("bar", mod = "DeadExt")
     )
     val result = compiler.compile(CircuitState(parse(input), ChirrtlForm, getAMap(annos)), Nil)
+    /* Uncomment to help debug
+    println(result.circuit.serialize)
+    result.annotations.get.annotations.foreach{ a =>
+      a match {
+        case DeletedAnnotation(xform, anno) => println(s"$xform deleted: ${a.target}")
+        case Annotation(target, _, _) => println(s"not deleted: $target")
+      }
+    }
+    */
```

Comment:
Do you really think this is so likely to break we need to keep the printlns around? We could also make it logger.debug stuff.